src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -1792,8 +1792,7 @@ public SVNAuthentication createSVNAuthentication(String kind) {
 
             @Override
             public StandardCredentials toCredentials(String description) {
-                return new UsernamePasswordCredentialsImpl(CredentialsScope.GLOBAL, null, description, userName,
-                        getPassword());
+                return new UsernamePasswordCredentialsImpl(CredentialsScope.GLOBAL, null, description, userName, getPassword());
```

Comment:
Such changes should be submitted as additional PRs, but actually it's unnecessary at all. The coding style does not explicitly define this requirement, and the text readability depends on your IDE window width.